src/modules/inventory/src/inventory.cpp

```diff
@@ -1,11 +1,13 @@
-#include <iostream>
+#include <inventory.hpp>
+
 #include <cjson/cJSON.h>
+#include <config.h>
 #include <defs.h>
 #include <logger.hpp>
-
-#include <inventory.hpp>
 #include <sysInfo.hpp>
 
+#include <iostream>
```

Comment:
It seems like this header is not needed.

Reply:
Removed